project-api/project-api-test/src/test/java/io/ballerina/projects/test/plugins/CompilerPluginTests.java

```diff
@@ -146,6 +147,29 @@ public void testInBuiltCompilerPlugin() throws IOException {
         Assert.assertTrue(logFileContent.contains(diagnosticResult.errors().stream().findFirst().get().toString()));
     }
 
+    @Test(description = "Test `pkg-toml-semantic-analyzer` compiler plugin by checking invalid export "
+            + "modules in `Ballerina.toml`")
+    public void testPkgInvalidExportedModule() {
+        Package currentPackage = loadPackage("package_invalid_exported_modules");
+        DiagnosticResult diagnosticResult = currentPackage.getCompilation().diagnosticResult();
+        Assert.assertEquals(diagnosticResult.diagnosticCount(), 3, "Unexpected number of compilation diagnostics");
+        Iterator<Diagnostic> diagnosticIterator = diagnosticResult.diagnostics().iterator();
+        Assert.assertEquals(diagnosticIterator.next().toString(), "ERROR [Ballerina.toml:(7:0,7:64)] "
+                + "could not locate dependency path '../libs/ballerina-runtime-api-2.0.0-beta.2-SNAPSHOT.jar'");
+        Assert.assertEquals(diagnosticIterator.next().toString(), "ERROR [Ballerina.toml:(4:10,4:15)] "
```

Comment:
Is the Toml location correct ? Shall we validate the location is the test as well ?

Reply:
<img width="276" alt="Screenshot 2021-07-12 at 10 51 49" src="https://user-images.githubusercontent.com/19742678/125234885-35acff80-e2ff-11eb-9ce2-4242d0a0f0c9.png">